decimal.go

```diff
@@ -27,6 +27,27 @@ func init() {
 	newDecimal.MarshalJSONWithoutQuotes = true
 }
 
+// Zero create a new decimal number that is equal to zero.
+func Zero() Number {
+	return newDecimal.New(0, 0)
```

Comment:
I think we can return `newDecimal.Zero` here.

Reply:
`newDecimal.Zero` actually has an exponent of `1`. It is more convenient for us to have `0` because that is what the previous version had.